starter/cloudrun-graphql/src/app/api/graphql/schema/schema.graphql

```diff
@@ -0,0 +1,3 @@
+type Greeting {
```

Comment:
Type that is not being queried anywhere or mutated anywhere. We need Query type instead. This is also a reason why the app is not starting, the GraphQL schema needs to have at least one query. Did you try to run the app?

```
Error: Query root type must be provided.
```

Feel free to reach me, we can quickly discuss or [read the GraphQL spec](https://graphql.org/learn/schema/).

Reply:
🤝